src/shared/DotnetTool/dotnet-tool.nuspec

```diff
@@ -1,7 +1,7 @@
 
 <package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
     <metadata>
-        <id>GitCredentialManager.Cli</id>
+        <id>git-credential-manager</id>
         <version>$version$</version>
         <description>Secure, cross-platform Git credential storage with authentication to Azure Repos, GitHub, and other popular Git hosting services.</description>
         <authors>Git Credential Manager</authors>
```

Comment:
Does the authors (or owners?) field need to match the NuGet.org org name?

Reply:
I don't believe so

Reply:
I don't think it needs to, but I think it's a good idea. Apparently [owners](https://learn.microsoft.com/en-us/nuget/reference/nuspec#optional-metadata-elements) is deprecated in favor of authors.